packages/flutter_tools/templates/plugin_swift_package_manager/ios-swift.tmpl/projectName.tmpl/Package.swift.tmpl

```diff
@@ -15,10 +15,14 @@ let package = Package(
     targets: [
         .target(
             name: "{{projectName}}",
-            dependencies: [],
-            resources: [
-               .process("Resources"),
-            ]
+            dependencies: []
+            // If your plugin requires a privacy manifest, for example if it uses any required
+            // reason APIs, update the PrivacyInfo.xcprivacy file to describe your plugin's privacy
+            // impact, and then uncomment these lines. For more information, see
+            // https://developer.apple.com/documentation/bundleresources/privacy_manifest_files
+            // , resources: [
```

Comment:
This is gross, but because the parameters are order-specific and trailing commas aren't allowed, I couldn't see any other option.

I tried using `excludes` so this wouldn't be conditional at all, but that caused build failures for some reason (maybe due to everything being excluded?)

Reply:
Also, the `Resources` folder is also where images and other resource files go, the comment kind of seems like it's only used for privacy manifests.

Maybe change to something like
"If your plugin uses resource files or requires a privacy manifest, uncomment the line below. If your plugin does not require a privacy manifest, delete the PrivacyInfo.xcprivacy file provided in the Resources directory. If your plugin does requires a privacy manifest, for example if it uses any required reason APIs, update the PrivacyInfo.xcprivacy file to describe your plugin's privacy impact. For more information, see https://developer.apple.com/documentation/bundleresources/privacy_manifest_files"

Idk, that's kind of long, though. We could also put the privacy manifest in a different spot so it's independent of other Resources

Reply:
So hopefully it's a niche situation they would have anything else in resources.  I don't have a preference.

Reply:
> Could put the comment inside and just comment out the `.process("Resources"),` line.

I thought I had tried that and gotten the same error as when I did exclude, but I must be misremembering since it works for me now.



> Also, the `Resources` folder is also where images and other resource files go, the comment kind of seems like it's only used for privacy manifests.

Yes, I'd thought about that too, but when I had tried to make the comment cover everything it seemed like kind of a mess.

> We could also put the privacy manifest in a different spot so it's independent of other Resources

Ooh, I like that. I hadn't considered that option; I'll set it up that way (and re-test to be sure it's working).

Do we want to keep the empty `Resources` folder at that point? I would be tempted to just put a comment saying other files or directories could be added if they happened to need other resources, since I don't think it's common.

Reply:
Updated to unconditionally have `resources:`, and moved the privacy manifest to a stand-alone file that is added to resources by a single commented-out line.

I also removed `Resources` and just put a comment since:
- I think having other resources will be rare,
- there's now an example right there someone can refer to if they need to see how it works, and
- it completely avoids the whole .gitkeep issue with pub.